### PR TITLE
Add package MentaLiST-0.1.2

### PR DESCRIPTION
**Pull request by dfornika**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dfornika** [comment]
I know that there are lots of recipes waiting to be merged, but is there anything blocking this one? Is there anything I can do to speed up the process?

File: recipes/mentalist/post-link.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+ln -s $PREFIX/bin/MentaLiST.jl $PREFIX/bin/mentalist
```
Comment (bgruening) [inline recipes/mentalist/post-link.sh]: Why is this done post install and not in the build script?

Comment (dfornika) [reply recipes/mentalist/post-link.sh]: No good reason. Moved it to `build.sh`.

**Comment by dfornika** [comment]
Can I request that this recipe be merged?

**Comment by dfornika** [comment]
@bgruening is there any reason this recipe can't be merged?

**Comment by bgruening** [comment]
No, the only reason is an awful slow Bjoern.

**Comment by dfornika** [comment]
No worries! It looks like there's a lot of work being managed by a small team. Thanks for your help with this.

**Pull request merged**